Acornfile

```diff
@@ -0,0 +1,120 @@
+name:        "Mastodon Acorn"
+description: "Running Masotdon on Acorn"
+readme:      "./README.md"
+icon:        "./mastodon.svg"
+
+
+services: postgres: {
+	image: "ghcr.io/acorn-io/postgres:v#.#-#"
+}
+
+services: redis: image: "ghcr.io/slayer321/redis:v0.0.1"
```

Comment:
Here, I have added redis from my repo with the required changes. Will update it once we have the required changed merge on the offical redis image.

Reply:
Can you create an issue for it, if not present already?